Marlin/configuration_store.cpp

```diff
@@ -1315,8 +1315,8 @@ void MarlinSettings::reset() {
         SERIAL_ECHOLNPGM("UBL_MESH_MAX_X     " STRINGIFY(UBL_MESH_MAX_X));
         SERIAL_ECHOLNPGM("UBL_MESH_MAX_Y     " STRINGIFY(UBL_MESH_MAX_Y));
 
-        SERIAL_ECHOLNPGM("MESH_X_DIST        " STRINGIFY(MESH_X_DIST));
-        SERIAL_ECHOLNPGM("MESH_Y_DIST        " STRINGIFY(MESH_Y_DIST));
+        SERIAL_ECHOLNPAIR("MESH_X_DIST        ", MESH_X_DIST);
+        SERIAL_ECHOLNPAIR("MESH_Y_DIST        ", MESH_Y_DIST);
```

Comment:
Yes it does save some FLASH.  Converting these 8 saves 208 bytes (26 each) with no change in RAM usage.

There's only 6 - 12 more candidates for this reduction.